.github/workflows/rspec_tests.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: RSpec test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  rspec_tests:
+    name: ${{ matrix.cfg.os }}(ruby ${{ matrix.cfg.ruby }})
+    strategy:
+      matrix:
+        cfg:
+          - {os: ubuntu-18.04, ruby: '2.5'}
+          - {os: ubuntu-18.04, ruby: '2.7'}
+          - {os: windows-2019, ruby: '2.5'}
+          - {os: windows-2019, ruby: '2.7'}
+
+    runs-on: ${{ matrix.cfg.os }}
+    steps:
+      - name: Checkout current PR
+        uses: actions/checkout@v2
+
+      - name: Install ruby version ${{ matrix.cfg.ruby }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.cfg.ruby }}
+      
```

Comment:
I'm guessing we need to `bundle install` after installing ruby and before running `bundle exec rake spec`? Puppet's version of this does: https://github.com/puppetlabs/puppet/blob/4772afa194402a3876069785a178611797a8eb7d/.github/workflows/checks.yaml#L32-L36

where the `gem update ... ` is necessary on Ruby 2.5 but we do it for all rubies.